dataland-frontend/tests/e2e/fixtures/lksg/LksgDataFixtures.ts

```diff
@@ -335,7 +351,7 @@ export function generateLksgData(undefinedProbability = 0.5): LksgData {
     environmental: {
       useOfMercuryMercuryWasteMinamataConvention: {
         mercuryAndMercuryWasteHandling: valueOrUndefined(randomYesNo(), undefinedProbability),
-        mercuryAndMercuryWasteHandlingPolicy: valueOrUndefined(randomYesNo(), undefinedProbability),
```

Comment:
Is undefined still possible or do we not test this case anymore?

Reply:
It can still be undefined with the function `generateBaseDataPointOrUndefined`

Reply:
ok